src/v2/recommender.py

```diff
@@ -462,6 +461,8 @@ def execute(self, arguments=None, persist=True, check_license=False):
         ended_at = datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%S.%f")
         audit = {'started_at': started_at, 'ended_at': ended_at, 'version': 'v2'}
 
+        recommendation = StackRecommendationResult(**recommendation,
```

Comment:
Why is this required now and not earlier?

Reply:
Actually I had missed this, but there is no in missing this. I added now just have a proper response validation.